src/test/scripts/elasticsearch-values.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I think this could be moved to `/src/test/infrastructure` so it is next to `nfs-server` helm chart. Arguably we should move the `postgres-values.yaml` there as well.